lib/srv/desktop/rdp/rdpclient/Cargo.toml

```diff
@@ -17,11 +17,8 @@ libc = "0.2.121"
 log = "0.4.16"
 num-derive = "0.3.3"
 num-traits = "0.2.14"
-# Ideally, we'd use RustCrypto/RSA instead of linking OpenSSL. Unfortunately,
-# RustCrypto doesn't expose the low-level primitives we need for the smartcard
-# challenge signing (see src/piv.rs for details).
-openssl = { version = "0.10.38", features = ["vendored"] }
 rand = { version = "0.8.5", features = ["getrandom"] }
 rand_chacha = "0.3.1"
-rdp-rs = { git = "https://github.com/gravitational/rdp-rs", rev = "f51222e7edeaf2954e2c02db3506aa36e02304ca" }
+rsa = { version = "0.6.1" }
+rdp-rs = { git = "https://github.com/gravitational/rdp-rs", rev = "30053f2d5b47542d4b878acff7b9e062a6046499" }
```

Comment:
Don't merge this PR until the rdp-rs changes land in master, then update the `rev` here.

Reply:
No worries! That was my plan to wait until we finalize the rdp-rs changes :)